<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginW</class>
 <widget class="QDialog" name="LoginW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>欢迎登录丑团外卖</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/new/prefix1/res/登录.png</normaloff>:/new/prefix1/res/登录.png</iconset>
  </property>
  <widget class="QLineEdit" name="editUser">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>190</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="inputMask">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="editPass">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>250</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 5px;/*设置圆角的大小*/
padding: 0.8px;/*如果没有内容光标在开始往后移动0.8像素点*/
selection-background-color: darkgray;
background:rgba(255, 255, 255, 200);</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnSureLogin">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>340</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>901</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(255, 255, 255, 200);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#ffffff;&quot;&gt;UglyTakeoutFood &lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ffffff;&quot;&gt;丑团外卖&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>747</x>
     <y>140</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;登录&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>310</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="rememberPass">
      <property name="styleSheet">
       <string notr="true">color: rgb(255,255,255);</string>
      </property>
      <property name="text">
       <string>记住密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labForget">
      <property name="maximumSize">
       <size>
        <width>95</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;忘记密码?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>430</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="labNonAccount">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;没有账号?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labRegister">
      <property name="font">
       <font>
        <family>Arial</family>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#24bf3e;&quot;&gt;立即注册&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>editUser</tabstop>
  <tabstop>editPass</tabstop>
  <tabstop>rememberPass</tabstop>
  <tabstop>BtnSureLogin</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
